### PR TITLE
contrib/tmux: new package (3.3a)

### PR DESCRIPTION
**Pull request by Bridouz**
- Built and tested working on aarch64 (rpi400)

### CODE REVIEW

File: contrib/tmux/template.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+makedepends = ["libevent-devel", "ncurses-devel" ]
+pkgdesc = "Terminal multiplexer"
+maintainer = "Justin Berthault <justin.berthault@zaclys.net>"
+license = "ISC"
+url = "https://tmux.github.io"
+source = f"https://github.com/tmux/tmux/releases/download/{pkgver}/tmux-{pkgver}.tar.gz"
+sha256 = "e4fd347843bd0772c4f48d6dde625b0b109b7a380ff15db21e97c11a4dcdf93f"
+
+def pre_configure(self):
+	self.do("autoreconf", "-if")
```
Comment (q66) [inline contrib/tmux/template.py:16]: also there should be a post_install with `self.install_license("LICENSE_FILE")`

File: contrib/tmux/template.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+pkgname = "tmux"
+pkgver = "3.3a"
+pkgrel = 0
+build_style = "configure"
```
Comment (q66) [inline contrib/tmux/template.py:4]: also this should be gnu_configure

File: contrib/tmux/template.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+hostmakedepends = ["byacc", "automake", "pkgconf" ]
+makedepends = ["libevent-devel", "ncurses-devel" ]
+pkgdesc = "Terminal multiplexer"
+maintainer = "Justin Berthault <justin.berthault@zaclys.net>"
+license = "ISC"
+url = "https://tmux.github.io"
+source = f"https://github.com/tmux/tmux/releases/download/{pkgver}/tmux-{pkgver}.tar.gz"
+sha256 = "e4fd347843bd0772c4f48d6dde625b0b109b7a380ff15db21e97c11a4dcdf93f"
+
+def pre_configure(self):
```
Comment (q66) [inline contrib/tmux/template.py:15]: this hook probably shouldn't be needed as the release comes with pre-generated autotools files

**Comment by Bridouz** [comment]
All good (thanks for the hints)

File: contrib/tmux/template.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+makedepends = ["libevent-devel", "ncurses-devel"]
+pkgdesc = "Terminal multiplexer"
+maintainer = "Justin Berthault <justin.berthault@zaclys.net>"
+license = "ISC"
+url = "https://tmux.github.io"
+source = f"https://github.com/tmux/tmux/releases/download/{pkgver}/tmux-{pkgver}.tar.gz"
+sha256 = "e4fd347843bd0772c4f48d6dde625b0b109b7a380ff15db21e97c11a4dcdf93f"
+
+def post_install(self):
+	self.install_license("COPYING")
```
Comment (q66) [inline contrib/tmux/template.py:16]: replace tab with 4 spaces

**Comment by q66** [comment]
merged, i also moved it to main/

**Pull request closed without merging**